yat-lib/src/main/java/yat/android/ui/components/CustomFont.kt

```diff
@@ -30,21 +30,20 @@
  * OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package yat.android.ui.component
+package yat.android.ui.components
 
 import android.content.Context
 import android.graphics.Typeface
 import java.util.*
 
-enum class CustomFont(private val fileName: String) {
+internal enum class CustomFont(private val fileName: String) {
 
-    ALLIANCE_BOLD("fonts/AllianceNo1-Bold.otf"),
-    ALLIANCE_SEMI_BOLD("fonts/AllianceNo1-SemiBold.otf"),
-    AVENIR_LT_STD_BLACK("fonts/AvenirLTStd-Black.otf"),
-    AVENIR_LT_STD_HEAVY("fonts/AvenirLTStd-Heavy.otf"),
-    AVENIR_LT_STD_MEDIUM("fonts/AvenirLTStd-Medium.otf"),
-    AVENIR_NEXT_LT_PRO_REGULAR("fonts/AvenirNextLTPro-Regular.otf"),
-    CIRCULAR_SPOTIFY_BOOK("fonts/CircularSpotifyTxT-Book.otf");
+    ALLIANCENO1_BOLD("fonts/AllianceNo1_Bold.ttf"),
+    ALLIANCENO1_BLACK("fonts/AllianceNo1_Black.ttf"),
+    ALLIANCENO1_LIGHT("fonts/AllianceNo1_Light.ttf"),
+    ALLIANCENO1_MEDIUM("fonts/AllianceNo1_Medium.ttf"),
+    ALLIANCENO1_REGULAR("fonts/AllianceNo1_Regular.ttf"),
+    ALLIANCENO1_SEMI_BOLD("fonts/AllianceNo1_SemiBold.ttf");
```

Comment:
Fonts should be provided by the developers which will integrate our framework into their product. If they will not be sepecified then our framework should use system font. Please check [YatStyle.swift](https://github.com/yat-labs/yat-lib-ios/blob/main/YatLib/Classes/Integration/YatStyle.swift) for more details.